content/developer/reference/backend/testing.rst

```diff
@@ -681,31 +725,73 @@ To start a tour from a python test, make the class inherit from
 Debugging tips
 --------------
 
-Running the tour in debug mode
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Observing tours in a browser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two ways with different tradeoffs:
+
+``watch=True``
+''''''''''''''
+When running a tour locally via the test suite, the ``watch=True``
+parameter can be added to the ``browser_js`` or ``start_tour``
+call::
 
-First, activate the :doc:`developer mode </applications/general/developer_mode>` with
-`?debug=tests`. Then, open your debug menu and click on **Start Tour**. You can now launch your tour
-from there with the button `Test`.
+    self.start_tour("/web", code, watch=True)
 
-You can also add this step in your tour to stop it right where you want it to:
+This will automatically open a Chrome window with the tour being
+run inside it.
+
+**Advantages**
+  - always works if the tour has Python setup / surrounding code, or multiple steps
+  - runs entirely automatically (just select the test which launches the tour)
+  - transactional (*should* always be runnable multiple times)
+**Drawbacks**
+  - only works locally
+  - only works if the test / tour can run correctly locally
+
+Run via browser
+'''''''''''''''
+Tours can also be launched via the browser UI, either by calling
 
 .. code-block:: javascript
 
-   {
-       trigger: "body",
-       run: () => {debugger}
-   }
+    odoo.startTour(tour_name);
+
+in the javascript console, or by enabling ``?debug=tests``, then
+selecting **Start Tour** in the debug menu and picking a tour:
+
+.. image:: testing/tours.png
```

Comment:
The image is rather small in full size and is not easy to read. Would you mind taking a higher resolution screenshot and scaling it down if needed? The image modal will allow reading the details better.